src/main/scala/com/jakainas/functions/package.scala

```diff
@@ -131,6 +131,10 @@ package object functions {
     def save()(implicit table: Table[T]): Unit = {
       ds.write.partitionBy(table.partitioning: _*).parquet(table.fullPath)
     }
+
+    def frequency (col: String, otherColumns: String*): DataFrame = {
```

Comment:
- Can you add some documentation for the function
- It should support columns as well as strings.